test/dynamo/test_base_hop.py

```diff
@@ -73,6 +74,140 @@ def forward(self, l_x_: "f32[3, 3]", l_y_: "f32[3, 3]"):
 """,  # NOQA: B950
         )
 
+    def _find_hop_schema(
+        self, gm: torch.fx.GraphModule, target: Any
+    ) -> list[torch._C.FunctionSchema]:
+        import torch.utils._pytree as pytree
+
+        schemas = []
+        for node in gm.graph.find_nodes(op="call_function", target=target):
+
+            def _get_example_value(node: torch.fx.Node) -> Any:
+                if node.op == "get_attr":
+                    return getattr(gm, node.target)
+                else:
+                    return node.meta["example_value"]
+
+            fake_args, fake_kwargs = pytree.tree_map_only(
+                torch.fx.Node,
+                _get_example_value,
+                (node.args, node.kwargs),
+            )
+            schema = node.target.gen_schema(*fake_args, **fake_kwargs)
+            schemas.append(schema)
+        return schemas
+
+    @torch._dynamo.config.patch(assume_static_by_default=True)
+    def test_schema_gen_single_return(self):
```

Comment:
Do you have a test with:
1) a None input?
2) an int input?

I would expect those to be treated as constants and not shown in the graph but just want to check

Reply:
Yeah, added tests to show this.